README.md

```diff
@@ -1,3 +1,22 @@
 # police-data-trust
+# Installation
+
+1. Make sure that you have [Python 3.8](https://www.python.org/), [Git](https://git-scm.com/), and [PostgreSQL](https://www.postgresql.org/) are installed on your machine.
+
+2. Clone this repository `git clone git@github.com:codeforboston/police-data-trust.git`
```

Comment:
Fork first.

Reply:
Making this change, and all previously suggested changes in the PR now.